ngraph/frontend/generic/src/frontend_manager.cpp

```diff
@@ -441,14 +490,17 @@ namespace ngraph
                 throw "Framework " + framework + " is unknown for FrontEnd manager; cannot load it.";
         }
 
-        FrontEnd::Ptr FrontEndManager::loadByModel (const std::string& path, FrontEndCapabilities fec)
+        FrontEnd::Ptr FrontEndManager::loadByModel(const std::string& path, FrontEndCapabilities fec)
         {
-            FRONT_END_NOT_IMPLEMENTED(loadByModel);
+            return m_impl->loadByModel(path, fec);
         }
 
-        std::vector<std::string> FrontEndManager::availableFrontEnds () const
+        std::vector<std::string> FrontEndManager::availableFrontEnds() const
         {
-            return {"onnx", "pdpd", "tf"};
+            if (m_impl) {
+                std::cout << "No impl";
```

Comment:
Are you sure?

Reply:
Very sorry, debugging code (had a problems with python bindings)

Reply:
Can you fix it and we will merge, I think.

Reply:
Fixed